docs/rules/key-spacing.md

```diff
@@ -121,53 +138,53 @@ var obj = {
 };
 ```
 
-### `"align": "colon"`
-
-The `align` option can also vertically align colons and values together. Whereas with `"value"` alignment, padding belongs right of the colon, with `"colon"` alignment, padding goes to the left of the colon. Except in the case of padding, it still respects `beforeColon` and `afterColon`. As with `"value"` alignment, groups of properties separated by blank lines are considered distinct and can have different alignment than other groups.
-
-The following patterns are considered valid:
+Examples of **correct** code for this rule with the `{ "align": "value" }` option:
 
 ```js
-/*eslint key-spacing: ["error", { "align": "colon" }]*/
+/*eslint key-spacing: ["error", { "align": "value" }]*/
 
 var obj = {
-    foobar   : 42,
-    bat      : (2 * 2),
-    "default": fn(),
+    a:    value,
+    bcde: 42,
 
-    fn : function() {},
-    abc: value
+    fg: foo(),
+    h:  function() {
+        return this.a;
+    },
+    ijkl: 'Non-consecutive lines form a new group'
 };
+
+var obj = { a: "foo", longPropertyName: "bar" };
 ```
 
+Examples of **incorrect** code for this rule with the `{ "align": "colon" }` option:
+
 ```js
-/*eslint key-spacing: ["error", { "align": "colon", "beforeColon": true, "afterColon": false }]*/
+/*eslint key-spacing: ["error", { mode": "minimum" }]*/
 
-obj = {
-    first  :1,
-    second :2,
-    third  :3
-};
+call({
+    foobar: 42,
+    bat:    (2 * 2)
+});
 ```
 
-The following patterns are considered problems:
+Examples of **correct** code for this rule with the `{ "align": "colon" }` option:
 
 ```js
-/*eslint key-spacing: ["error", { "align": "colon" }]*/
+/*eslint key-spacing: ["error", { mode": "minimum" }]*/
 
-var obj = {
-    one:   1,
-    "two": 2,
-    three:  3
-};
+call({
+    foobar: 42,
+    bat   : (2 * 2)
+});
 ```
 
-### Fine-grained control
+### singleLine and multiLine
 
-You can specify these options separately for single-line and multi-line configurations by organizing the options this way:
+Examples of **correct** code for this rule with the `{ "singleLine": { }, "multiLine": { } }` options:
```

Comment:
If you mean as you did here with h3 singleLine and multiLine 👍 when they are so interrelated